docs/coding-guidelines/package-projects.md

```diff
@@ -216,7 +216,7 @@ Sample `System.IO.FileSystem.pkgproj`
 ## Asset selection
```

Comment:
Fix [MD022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines).

```suggestion
## Asset selection

```